pkg/sql/logictest/testdata/logic_test/geospatial_zm

```diff
@@ -83,3 +83,33 @@ INSERT INTO geom_4d VALUES ('point(1 2)')
 
 statement error pq: object type PointZ does not match column dimensionality GeometryZM
 INSERT INTO geom_4d VALUES ('pointz(1 2 3)')
+
+# Point specific operations
+query RRRRI
+SELECT
+  ST_X(a.geom),
+  ST_Y(a.geom),
+  ST_Z(a.geom),
+  ST_M(a.geom),
+  ST_Zmflag(a.geom)
```

Comment:
super nit: i'd call this `ST_ZMFlag`

Reply:
I think PostGIS renamed it to `ST_Zmflag` https://postgis.net/docs/ST_Zmflag.html. Do you still want it to be `ST_ZMFlag`?